CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# Contributing to Eclipse Tractus-X
+
+Thanks for your interest in this project.
+
+## Project description
+
+The companies involved want to increase the automotive industry's
+competitiveness, improve efficiency through industry-specific cooperation and
+accelerate company processes through standardization and access to information
+and data. A special focus is also on SMEs, whose active participation is of
+central importance for the network's success. That is why Catena-X has been
+conceived from the outset as an open network with solutions ready for SMEs,
+where these companies will be able to participate quickly and with little IT
+infrastructure investment. Tractus-X is meant to be the PoC project of the
+Catena-X alliance focusing on parts traceability.
+
+* https://projects.eclipse.org/projects/automotive.tractusx
+
+## Project licenses
+
+The Tractus-X project uses the following licenses:
+
+* Apache-2.0 for code
+* CC-BY-4.0 for non-code
+
+## Terms of Use
+
+This repository is subject to the Terms of Use of the Eclipse Foundation
+
+* https://www.eclipse.org/legal/termsofuse.php
+
+## Developer resources
+
+Information regarding source code management, builds, coding standards, and
+more.
+
+* https://projects.eclipse.org/projects/automotive.tractusx/developer
+
+Getting started:
+
+* https://eclipse-tractusx.github.io/docs/developer
+
+* https://projects.eclipse.org/projects/automotive.tractusx/developer
+
+The project maintains the source code repositories
+
+- https://github.com/eclipse-tractusx/ssi-dim-wallet-stub
+
+## Eclipse Development Process
+
+This Eclipse Foundation open project is governed by the Eclipse Foundation
+Development Process and operates under the terms of the Eclipse IP Policy.
+
+* https://eclipse.org/projects/dev_process
+* https://www.eclipse.org/org/documents/Eclipse_IP_Policy.pdf
+
+## Eclipse Contributor Agreement
+
+In order to be able to contribute to Eclipse Foundation projects you must
+electronically sign the Eclipse Contributor Agreement (ECA).
+
+* http://www.eclipse.org/legal/ECA.php
+
+The ECA provides the Eclipse Foundation with a permanent record that you agree
+that each of your contributions will comply with the commitments documented in
+the Developer Certificate of Origin (DCO). Having an ECA on file associated with
+the email address matching the "Author" field of your contribution's Git commits
+fulfills the DCO's requirement that you sign-off on your contributions.
+
+For more information, please see the Eclipse Committer Handbook:
+https://www.eclipse.org/projects/handbook/#resources-commit
+
+## How To Contribute
+
+For more practical information, please refer
+to [How to contribute.md](docs%2Fdeveloper%2FTechnical%20Documentation%2FDev%20Process%2FHow%20to%20contribute.md)
```

Comment:
due to the move of the documentation, this link is broken now

Reply:
Addressed with: https://github.com/eclipse-tractusx/ssi-dim-wallet-stub/pull/1/commits/eca1968a1351659f0f66cb3d0c285cd1640724bd